crates/holochain_types/src/app.rs

```diff
@@ -65,8 +65,13 @@ pub struct RegisterDnaPayload {
 /// The instructions on how to request NetworkInfo
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct NetworkInfoRequestPayload {
+    /// The calling agent
+    pub agent_pub_key: AgentPubKey,
     /// Get gossip info for these DNAs
     pub dnas: Vec<DnaHash>,
+    /// Timestamp in ms since which received amount of bytes from peers will
+    /// be returned. Defaults to UNIX_EPOCH.
+    pub last_time_queried: Option<u128>,
```

Comment:
use our `Timestamp` struct. It has microsecond precision while fitting in 64 bits, and is convention in our code.